src/fabtrimino.c

```diff
@@ -0,0 +1,250 @@
+#include "fabtrimino.h"
+
+struct square
+{
+    enum shape shape;
```

Comment:
This feels a bit pointless for a struct but I'm assuming you're going to extend it later.

Reply:
I indeed intend to extend it. It might be future proofing since that will only be necessary for the "square fabtris" mode, which will be added much later.